### PR TITLE
Updated the list view for the security group target.

### PR DESCRIPTION
**Pull request by BShrivastav**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/is_security_group_target.html.markdown [hunk 1]
```diff
@@ -9,10 +9,7 @@ description: |-
 
 # ibm_is_security_group_target
```
Comment (SunithaGudisagarIBM) [inline website/docs/r/is_security_group_target.html.markdown:10]: when checked in https://registry.terraform.io/tools/doc-preview tool,

Note

"IBM Cloud terraform provider currently provides both a standalone" portion has not come right, Can you please modify this

Comment (BShrivastav) [reply website/docs/r/is_security_group_target.html.markdown:10]: This is aligned to the above paragraph, do we really need this ?

Comment (SunithaGudisagarIBM) [reply website/docs/r/is_security_group_target.html.markdown:10]: Please check whether we need it or not, and remove if not required.
if required we need to add like   ~>**Note:**  section

Comment (sumitkumartiwari) [reply website/docs/r/is_security_group_target.html.markdown:10]: I think Note section is aligned as per other resources are using top section Note like BaaS.

Comment (sumitkumartiwari) [reply website/docs/r/is_security_group_target.html.markdown:10]: There is some alignment issues with provider section - please fix it.

Comment (BShrivastav) [reply website/docs/r/is_security_group_target.html.markdown:10]: done.

**Review by sumitkumartiwari — APPROVED** [review]

**Comment by sumitkumartiwari** [comment]
Please upstream the PR and close it

**Comment by BShrivastav** [comment]
Upstream raised: https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4247

**Pull request closed without merging**